src/cli-commands/update-dependencies.js

```diff
@@ -0,0 +1,98 @@
+const fs = require('fs-extra');
+const path = require('path');
+const spawn = require('cross-spawn');
+const logger = require('@blackbaud/skyux-logger');
+const latestVersion = require('latest-version');
+const rootPath = path.join(__dirname, '../../../../..');
```

Comment:
What's this? Could you use `process.cwd()` instead?

Reply:
Yeah I was really debating on this one, it seemed like process.cwd() wasn't reliable most of the time because it was based on wherever the user was trying to run the command.  Not sure how else I can get the main projects root directory reliably as the command doesn't pass in any useful arguments when it gets called.